Description: shared libpng
Index: ppsspp-1.6.3/Core/Screenshot.cpp
===================================================================
--- ppsspp-1.6.3.orig/Core/Screenshot.cpp
+++ ppsspp-1.6.3/Core/Screenshot.cpp
@@ -21,7 +21,7 @@
 #ifdef USING_QT_UI
 #include <QtGui/QImage>
 #else
-#include <libpng17/png.h>
+#include <libpng/png.h>
 #include "ext/jpge/jpge.h"
 #endif
 
Index: ppsspp-1.6.3/Core/TextureReplacer.cpp
===================================================================
--- ppsspp-1.6.3.orig/Core/TextureReplacer.cpp
+++ ppsspp-1.6.3/Core/TextureReplacer.cpp
@@ -16,7 +16,7 @@
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
 #ifndef USING_QT_UI
-#include <libpng17/png.h>
+#include <libpng/png.h>
 #endif
 
 #include <algorithm>
Index: ppsspp-1.6.3/ext/native/image/png_load.cpp
===================================================================
--- ppsspp-1.6.3.orig/ext/native/image/png_load.cpp
+++ ppsspp-1.6.3/ext/native/image/png_load.cpp
@@ -5,7 +5,7 @@
 #ifdef USING_QT_UI
 #include <QtGui/QImage>
 #else
-#include "libpng17/png.h"
+#include "libpng/png.h"
 #endif
 
 #include "png_load.h"
