Description: shared FFmpeg
Index: ppsspp-git/Qt/Settings.pri
===================================================================
--- ppsspp-git.orig/Qt/Settings.pri	2016-07-28 12:36:43.923705054 -0300
+++ ppsspp-git/Qt/Settings.pri	2016-07-28 12:36:43.915704943 -0300
@@ -32,12 +32,10 @@
 # Work out platform name
 include(Platform/OSDetection.pri)
 # OS dependent paths
-!system_ffmpeg: INCLUDEPATH += $$P/ffmpeg/$${PLATFORM_NAME}/$${PLATFORM_ARCH}/include
 
 !contains(CONFIG, staticlib) {
 	QMAKE_LIBDIR += $$CONFIG_DIR
-	!system_ffmpeg: QMAKE_LIBDIR += $$P/ffmpeg/$${PLATFORM_NAME}/$${PLATFORM_ARCH}/lib/
-	contains(DEFINES, USE_FFMPEG): LIBS+=  -lavformat -lavcodec -lavutil -lswresample -lswscale
+	contains(DEFINES, USE_FFMPEG): LIBS+=  -lavformat-ffmpeg -lavcodec-ffmpeg -lavutil-ffmpeg -lswresample-ffmpeg -lswscale-ffmpeg
 	equals(PLATFORM_NAME, "linux"):arm|android: LIBS += -lEGL
 }
 
Index: ppsspp-git/Core/HLE/sceAtrac.cpp
===================================================================
--- ppsspp-git.orig/Core/HLE/sceAtrac.cpp	2016-07-28 12:36:43.923705054 -0300
+++ ppsspp-git/Core/HLE/sceAtrac.cpp	2016-07-28 12:36:43.915704943 -0300
@@ -119,9 +119,9 @@
 #ifdef USE_FFMPEG
 
 extern "C" {
-#include "libavformat/avformat.h"
-#include "libswresample/swresample.h"
-#include "libavutil/samplefmt.h"
+#include <libavformat/avformat.h>
+#include <libswresample/swresample.h>
+#include <libavutil/samplefmt.h>
 }
 
 #endif // USE_FFMPEG
Index: ppsspp-git/Core/HLE/sceMpeg.cpp
===================================================================
--- ppsspp-git.orig/Core/HLE/sceMpeg.cpp	2016-07-28 12:36:43.923705054 -0300
+++ ppsspp-git/Core/HLE/sceMpeg.cpp	2016-07-28 12:36:43.915704943 -0300
@@ -98,9 +98,9 @@
 #ifdef USE_FFMPEG 
 
 extern "C" {
-#include "libavformat/avformat.h"
-#include "libavutil/imgutils.h"
-#include "libswscale/swscale.h"
+#include <libavformat/avformat.h>
+#include <libavutil/imgutils.h>
+#include <libswscale/swscale.h>
 }
 static AVPixelFormat pmp_want_pix_fmt;
 
Index: ppsspp-git/Core/HW/MediaEngine.cpp
===================================================================
--- ppsspp-git.orig/Core/HW/MediaEngine.cpp	2016-07-28 12:36:43.923705054 -0300
+++ ppsspp-git/Core/HW/MediaEngine.cpp	2016-07-28 12:36:43.915704943 -0300
@@ -32,10 +32,10 @@
 
 extern "C" {
 
-#include "libavcodec/avcodec.h"
-#include "libavformat/avformat.h"
-#include "libavutil/imgutils.h"
-#include "libswscale/swscale.h"
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
+#include <libavutil/imgutils.h>
+#include <libswscale/swscale.h>
 
 }
 #endif // USE_FFMPEG
Index: ppsspp-git/Core/HW/SimpleAudioDec.cpp
===================================================================
--- ppsspp-git.orig/Core/HW/SimpleAudioDec.cpp	2016-07-28 12:36:43.923705054 -0300
+++ ppsspp-git/Core/HW/SimpleAudioDec.cpp	2016-07-28 12:36:43.915704943 -0300
@@ -26,9 +26,9 @@
 #ifdef USE_FFMPEG
 
 extern "C" {
-#include "libavformat/avformat.h"
-#include "libswresample/swresample.h"
-#include "libavutil/samplefmt.h"
+#include <libavformat/avformat.h>
+#include <libswresample/swresample.h>
+#include <libavutil/samplefmt.h>
 }
 
 #endif  // USE_FFMPEG
