Index: ppsspp-git/ffmpeg/linux_arm.sh
===================================================================
--- ppsspp-git.orig/ffmpeg/linux_arm.sh	2015-10-04 17:22:11.740735228 -0300
+++ ppsspp-git/ffmpeg/linux_arm.sh	2015-10-04 17:28:36.730644290 -0300
@@ -1,13 +1,5 @@
 #!/bin/bash
 
-GENERAL="\
-   --enable-cross-compile \
-   --extra-libs="-lgcc" \
-   --arch=arm \
-   --cc=arm-linux-gnueabi-gcc \
-   --cross-prefix=arm-linux-gnueabi- \
-   --nm=arm-linux-gnueabi-nm"
-
 MODULES="\
    --disable-avdevice \
    --disable-filters \
@@ -99,8 +91,7 @@
 {
 ./configure --target-os=linux \
     --prefix=./linux/armv7 \
-    ${GENERAL} \
-    --extra-cflags=" -O3 -fasm -Wno-psabi -fno-short-enums -fno-strict-aliasing -finline-limit=300 -mfloat-abi=softfp -mfpu=neon -marm -march=armv7-a" \
+    --extra-cflags=" -O3 -fasm -Wno-psabi -fno-short-enums -fno-strict-aliasing -finline-limit=300 -mfloat-abi=hard -mfpu=neon -marm -march=armv7-a" \
     --disable-shared \
     --enable-static \
     --enable-zlib \
@@ -117,6 +108,5 @@
 make install
 }
 
-build_ARMv6
 build_ARMv7
 echo Linux ARM builds finished
