Index: ppsspp-git2/CMakeLists.txt
===================================================================
--- ppsspp-git2.orig/CMakeLists.txt
+++ ppsspp-git2/CMakeLists.txt
@@ -129,6 +129,10 @@ if(NOT OPENGL_LIBRARIES AND USING_GLES2)
 	set(OPENGL_LIBRARIES GLESv2)
 endif()
 
+if(USING_EGL)
+	set(OPENGL_LIBRARIES ${OPENGL_LIBRARIES} EGL)
+endif()
+
 if(NOT OPENGL_LIBRARIES)
 	find_package(OpenGL REQUIRED)
 endif()
