Index: ppsspp-git/Qt/PPSSPP.pro
===================================================================
--- ppsspp-git.orig/Qt/PPSSPP.pro	2015-06-23 17:57:28.000000000 -0300
+++ ppsspp-git/Qt/PPSSPP.pro	2015-06-23 17:57:28.000000000 -0300
@@ -78,9 +78,7 @@
 }
 
 # UI
-SOURCES += $$P/UI/*.cpp \
-	$$P/android/jni/TestRunner.cpp
-arm:android: SOURCES += $$P/android/jni/ArmEmitterTest.cpp
+SOURCES += $$P/UI/*.cpp
 HEADERS += $$P/UI/*.h
 
 INCLUDEPATH += $$P $$P/Common $$P/native $$P/native/ext
Index: ppsspp-git/UI/CwCheatScreen.cpp
===================================================================
--- ppsspp-git.orig/UI/CwCheatScreen.cpp	2015-06-23 17:37:12.000000000 -0300
+++ ppsspp-git/UI/CwCheatScreen.cpp	2015-06-23 17:57:28.000000000 -0300
@@ -15,7 +15,9 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+#ifdef ANDROID
 #include "android/app-android.h"
+#endif
 #include "input/input_state.h"
 #include "ui/ui.h"
 #include "util/text/utf8.h"
Index: ppsspp-git/UI/EmuScreen.cpp
===================================================================
--- ppsspp-git.orig/UI/EmuScreen.cpp	2015-06-23 17:37:12.000000000 -0300
+++ ppsspp-git/UI/EmuScreen.cpp	2015-06-23 17:57:28.000000000 -0300
@@ -17,7 +17,9 @@
 
 #include <algorithm>
 
+#ifdef ANDROID
 #include "android/app-android.h"
+#endif
 #include "base/display.h"
 #include "base/logging.h"
 #include "base/timeutil.h"
Index: ppsspp-git/UI/GameSettingsScreen.cpp
===================================================================
--- ppsspp-git.orig/UI/GameSettingsScreen.cpp	2015-06-23 17:37:12.000000000 -0300
+++ ppsspp-git/UI/GameSettingsScreen.cpp	2015-06-23 17:58:11.546489231 -0300
@@ -45,7 +45,9 @@
 #include "Core/Host.h"
 #include "Core/System.h"
 #include "Core/Reporting.h"
+#ifdef ANDROID
 #include "android/jni/TestRunner.h"
+#endif
 #include "GPU/GPUInterface.h"
 #include "GPU/GLES/Framebuffer.h"
 
@@ -1069,7 +1071,9 @@
 }
 
 UI::EventReturn DeveloperToolsScreen::OnRunCPUTests(UI::EventParams &e) {
+#ifdef ANDROID
 	RunTests();
+#endif
 	return UI::EVENT_DONE;
 }
 
