Index: ppsspp-git/Core/Util/GameManager.cpp
===================================================================
--- ppsspp-git.orig/Core/Util/GameManager.cpp	2015-09-09 00:19:28.798937916 -0300
+++ ppsspp-git/Core/Util/GameManager.cpp	2015-09-09 00:24:18.474803566 -0300
@@ -20,7 +20,7 @@
 #include <cstring>
 
 #include "file/file_util.h"
-#include "ext/libzip/zip.h"
+#include <zip.h>
 #include "thread/thread.h"
 #include "util/text/utf8.h"
 
Index: ppsspp-git/Qt/Native.pro
===================================================================
--- ppsspp-git.orig/Qt/Native.pro	2015-09-09 00:19:28.798937916 -0300
+++ ppsspp-git/Qt/Native.pro	2015-09-09 00:24:43.426704692 -0300
@@ -54,10 +54,6 @@
 	HEADERS += $$P/ext/zlib/*.h
 }
 
-# Libzip
-SOURCES += $$P/ext/native/ext/libzip/*.c
-HEADERS += $$P/ext/native/ext/libzip/*.h
-
 # Native
 
 SOURCES +=  $$P/ext/native/audio/*.cpp \
Index: ppsspp-git/Qt/PPSSPP.pro
===================================================================
--- ppsspp-git.orig/Qt/PPSSPP.pro	2015-09-09 00:19:28.802937932 -0300
+++ ppsspp-git/Qt/PPSSPP.pro	2015-09-09 00:25:57.994348054 -0300
@@ -53,7 +53,7 @@
 	LIBS += -lGLEW
 }
 
-LIBS += -lsnappy
+LIBS += -lsnappy -lzip
 
 unix:contains(QT_CONFIG, system-zlib) {
 	LIBS += -lz
Index: ppsspp-git/ext/libkirk/cmd
===================================================================
--- ppsspp-git.orig/ext/libkirk/cmd	2015-09-09 00:19:28.798937916 -0300
+++ ppsspp-git/ext/libkirk/cmd	2015-09-09 00:25:28.122501728 -0300
@@ -54,7 +54,7 @@
 kirk/libkirk.a
 base/libtimeutil.a
 kirk/libkirk.a
-libzip/libzip.a
+-lzip
 kirk/libkirk.a
 base/libbase.a
 kirk/libkirk.a
@@ -84,7 +84,7 @@
 kirk/libkirk.a
 file/libfile.a
 kirk/libkirk.a
-libzip/libzip.a
+-lzip
 kirk/libkirk.a
 gfx/libgfx.a
 kirk/libkirk.a
