Index: ppsspp-git/native/gfx_es2/draw_text.h
===================================================================
--- ppsspp-git.orig/native/gfx_es2/draw_text.h	2014-08-26 20:27:28.665579000 -0300
+++ ppsspp-git/native/gfx_es2/draw_text.h	2015-06-18 23:21:54.685198589 -0300
@@ -8,6 +8,10 @@
 
 #pragma once
 
+#ifdef USING_QT_UI
+#include <QtGui/QFont>
+#endif
+
 #include <map>
 
 #include "base/basictypes.h"
@@ -19,10 +23,6 @@
 class Thin3DContext;
 class Thin3DTexture;
 
-#ifdef USING_QT_UI
-#include <QtGui/QFont>
-#endif
-
 struct TextStringEntry {
 	Thin3DTexture *texture;
 	int width;
