Index: ppsspp-git/native/gfx_es2/draw_text.h
===================================================================
--- ppsspp-git.orig/native/gfx_es2/draw_text.h	2015-06-19 11:36:25.846823903 -0300
+++ ppsspp-git/native/gfx_es2/draw_text.h	2015-06-19 11:36:25.846823903 -0300
@@ -8,6 +8,10 @@
 
 #pragma once
 
+#ifdef USING_QT_UI
+#include <QtGui/QFont>
+#endif
+
 #include <map>
 
 #include "base/basictypes.h"
@@ -19,10 +23,6 @@
 class Thin3DContext;
 class Thin3DTexture;
 
-#ifdef USING_QT_UI
-#include <QtGui/QFont>
-#endif
-
 struct TextStringEntry {
 	Thin3DTexture *texture;
 	int width;
Index: ppsspp-git/native/gfx_es2/draw_text.cpp
===================================================================
--- ppsspp-git.orig/native/gfx_es2/draw_text.cpp	2015-06-19 11:36:43.271067681 -0300
+++ ppsspp-git/native/gfx_es2/draw_text.cpp	2015-06-19 11:37:17.983592368 -0300
@@ -1,3 +1,9 @@
+#ifdef USING_QT_UI
+#include <QtGui/QImage>
+#include <QtGui/QPainter>
+#include <QtGui/QFontMetrics>
+#endif
+
 #include "base/logging.h"
 #include "base/stringutil.h"
 #include "thin3d/thin3d.h"
@@ -6,9 +12,6 @@
 #include "gfx_es2/draw_text.h"
 
 #ifdef USING_QT_UI
-#include <QtGui/QImage>
-#include <QtGui/QPainter>
-#include <QtGui/QFontMetrics>
 #include <QtOpenGL/QGLWidget>
 #endif
 
